tasks/main.yml

```diff
@@ -18,14 +18,10 @@
 - name: Download remote_syslog
   get_url: url="https://github.com/papertrail/remote_syslog2/releases/download/v{{ papertrail_version }}/remote_syslog_linux_amd64.tar.gz"
            dest="/usr/local/src/remote_syslog_{{ papertrail_version }}_linux_amd64.tar.gz"
+  register: remote_syslog_download
 
-- name: Extract remote_syslog
-  unarchive: src="/usr/local/src/remote_syslog_{{ papertrail_version }}_linux_amd64.tar.gz"
-             dest=/opt
-             copy=no
-             owner=root
-             group=root
-             creates=/opt/remote_syslog/remote_syslog
+- include: extract-remote-syslog.yml
```

Comment:
`ansible-lint` was throwing a "Tasks that run when changed should likely be handler" error (`ANSIBLE0016`). Using `include` satisfies the linter while keeping the desired behavior.